template/src/ontology/Makefile.jinja2

```diff
@@ -780,6 +780,13 @@ $(ONT)-basic.owl: $(SRC) $(OTHER_SRC) $(SIMPLESEED) $(KEEPRELATIONS)
 {% endif -%}
 {% endfor -%}
 
+# ----------------------------------------
+# Debugging Tools
+# ----------------------------------------
+
+explain_unsat: $(SRC) 
+	$(ROBOT) explain -i $(SRC) -M unsatisfiability --unsatisfiable random:10 --explanation tmp/$@.md
```

Comment:
Sorry one last, use $(TMP) (make sure this is the one) instead of `tmp`

Reply:
Done - I assume it will automatically make it a markdown file?

Reply:
Ah sorry, this would try to save the explanation as a  file named `$(TMP)` - that would not work. The explanation needs to be called `--explanation $(TMP)/$@.md`. If ROBOT sees the md, it understands that it should be markdown. Sorry about the confusion.

Reply:
yeah wasn't sure what $(TMP) pointed to, but got it now :) have made the changes. Thanks

Reply:
Thank you!